### ISSUE TITLE
proposal: add a way to get a specific entry from PhotoSize fields

### ISSUE THREAD

**Issue by gothchibjo** [tucnak/telebot#386]
It's important for me to get a thumbnail not larger than _width_ or _height_, so I would be glad to see something like Message.Photo.[]Sizes. Can I get it from the telebot?

I found in the docs:

    type Message struct {
    ...
    // For a photo, all available sizes (thumbnails).
    //              ^^^^^^^^^^^^^^^^^^^
    Photo *Photo `json:"photo"`

But the Photo struct contains no other thumbnail sizes except the largest one.

**Comment by demget** [comment]
You're right, currently, it's impossible to get any except the largest one. Maybe we should consider allowing it as well.

**Comment by ghazzor** [comment]
coming from python, i did  `message.photo[2].file_id` to get a lower quality image if the network speed is rather slow; (its a time sensitive bot and i run it locally on a weak net or multiple files sometimes); it would be really nice to have something like this.
